knowledge-base/common-kb-stackoverflowexception-editing-circular-references.md

```diff
@@ -0,0 +1,63 @@
+---
+title: How to Resolve StackOverflowException When Editing Items with Circular References in ListView for Blazor
```

Comment:
Since the KB article is about multiple components, we should either:
* Omit the specific component name in the title, because it can be misleading.
* Use the name of the most popular component.